pyseq.py

```diff
@@ -128,6 +139,24 @@ def __init__(self, item):
         self.head = self.name
         self.tail = ''
 
+    def __eq__(self, other):
```

Comment:
Maybe because they were different instances of the same file path, so their internal id's were different. In that case, implementing __eq__ makes sense.